src/metadata/runtime_bindings.rs

```diff
@@ -167,6 +176,44 @@ impl RuntimeBindingsMeta {
         )?)
     }
 
+    /// Build if necessary the extended euclidean algorithm used in circuit inverse gates
+    pub fn extended_euclidean_algorithm<'c, 'a>(
+        &mut self,
+        context: &'c Context,
+        module: &Module,
+        block: &'a Block<'c>,
+        location: Location<'c>,
+        rhs_value: Value<'c, '_>,
+        circuit_modulus: Value<'c, '_>,
```

Comment:
Given that this is a generic implementation of egcd, we could rename arguments to `a` and `b`.  (no need to keep the circuit-related names)

```suggestion
        rhs_value: Value<'c, '_>,
        circuit_modulus: Value<'c, '_>,
```